libs/blocks/notification/notification.js

```diff
@@ -72,7 +72,7 @@ function getBlockData(el) {
 function wrapCopy(foreground) {
   const text = foreground.querySelector('.text');
   if (!text) return;
-  const heading = text?.querySelector('h1, h2, h3, h4, h5, h6');
+  const heading = text?.querySelector('h1, h2, h3, h4, h5, h6') || text?.querySelector('p:not(.icon-area, .action-area)');
```

Comment:
Wouldn't this achieve the same result?
```suggestion
  const heading = text?.querySelector('h1, h2, h3, h4, h5, h6, p:not(.icon-area, .action-area)');
```